src/MCMCChains.jl

```diff
@@ -8,17 +8,14 @@ import Serialization: serialize, deserialize
 import Base: sort, range, names
 import Statistics: cor
 
-using RecipesBase
-import RecipesBase: plot
-
+using Requires
 using Serialization
 using Distributions
 using SpecialFunctions
 using AxisArrays
 const axes = Base.axes
 
 export Chains, getindex, setindex!, chains
-export plot, traceplot, meanplot, densityplot, histogramplot, mixeddensityplot, autcorplot
```

Comment:
all you have to do is delete this line - these should not be exported, they will become automagically available when Plots is loaded (you can also see that there is a typo in `autcorplot` which has of course had no effect)